pcsx2/gui/AppCoreThread.cpp

```diff
@@ -406,42 +403,41 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 	ResetCheatsCount();
 
 	//Till the end of this function, entry CRC will be 00000000
-	if (!gameCRC.Length()) {
-		if (EmuConfig.EnableWideScreenPatches || EmuConfig.EnableCheats) {
-			Console.WriteLn(Color_Gray, "Patches: No CRC, using 00000000 instead.");
-		}
+	if (!gameCRC.Length())
+	{
+		Console.WriteLn(Color_Gray, "Patches: No CRC found, using 00000000 instead.");
 		gameCRC = L"00000000";
 	}
 
 	// regular cheat patches
-	if (EmuConfig.EnableCheats) {
-		if (numberLoadedCheats = LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats")) {
-			gameCheats.Printf(L" [%d Cheats]", numberLoadedCheats);
-		}
-	}
+	if (EmuConfig.EnableCheats)
+		gameCheats.Printf(L" [%d Cheats]", LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats"));
 
 	// wide screen patches
-	if (EmuConfig.EnableWideScreenPatches) {
-		if (numberLoadedWideScreenPatches = LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks")) {
+	if (EmuConfig.EnableWideScreenPatches)
+	{
+		if ( int numberLoadedWideScreenPatches = LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks"))
 			gameWsHacks.Printf(L" [%d widescreen hacks]", numberLoadedWideScreenPatches);
-		}
-		else {
+		else
+		{
 			// No ws cheat files found at the cheats_ws folder, try the ws cheats zip file.
 			wxString cheats_ws_archive = Path::Combine(PathDefs::GetProgramDataDir(), wxFileName(L"cheats_ws.zip"));
 
-			if (numberDbfCheatsLoaded = LoadCheatsFromZip(gameCRC, cheats_ws_archive)) {
+			if ( int numberDbfCheatsLoaded = LoadCheatsFromZip(gameCRC, cheats_ws_archive))
+			{
 				PatchesCon->WriteLn(Color_Green, "(Wide Screen Cheats DB) Patches Loaded: %d", numberDbfCheatsLoaded);
 				gameWsHacks.Printf(L" [%d widescreen hacks]", numberDbfCheatsLoaded);
 			}
+			else
+			{
+				PatchesCon->WriteLn(Color_Green, "(Wide Screen Cheats DB) No Widescreen hacks were found for the following game");
+				gameWsHacks.Printf(L" [%d widescreen hacks]", numberDbfCheatsLoaded);
+			}
 		}
 	}
 
 	wxString consoleTitle = gameName + gameSerial;
-	if (!gameSerial.IsEmpty()) {
-		consoleTitle += L" [" + gameCRC.MakeUpper() + L"]";
-	}
-	consoleTitle += gameCompat + gameFixes + gamePatch + gameCheats + gameWsHacks;
-
+	consoleTitle += L" [" + gameCRC.MakeUpper() + L"]" + gameCompat + gameFixes + gamePatch + gameCheats + gameWsHacks;
```

Comment:
Hmm. Now that I look at it, it is quite long :P

Up to you then whether you split the statement back into two lines or not.